code/modules/hydroponics/grown/cannabis.dm

```diff
@@ -89,7 +89,7 @@
 	seed = /obj/item/seeds/cannabis
 	icon = 'icons/goonstation/objects/hydroponics.dmi'
 	name = "cannabis leaf"
-	desc = "Recently legalized in most galaxies."
+	desc = "A common marijuana leaf. Its legality varies heavily across Sector polities."
```

Comment:
The word "sector" should not be capitalized, as it's just a definition of space. Unless you are refering to the "Orion sector" in particular, in which case we would have to decide what official name to use to the known space once and for all.
```suggestion
	desc = "A common marijuana leaf. Its legality varies heavily across sector polities."
```